docs/reference-guides/rancher-security/rancher-v2.6-hardening-guides/rke1-self-assessment-guide-with-cis-v1.23-benchmark.md

```diff
@@ -0,0 +1,2601 @@
+```
+
+**Returned Value**:
+
+```console
+root 13376 13354 13 10:27 ? 00:00:56 kube-apiserver --runtime-config=authorization.k8s.io/v1beta1=true --requestheader-username-headers=X-Remote-User --cloud-provider= --service-cluster-ip-range=10.43.0.0/16 --requestheader-group-headers=X-Remote-Group --storage-backend=etcd3 --audit-log-maxage=30 --audit-policy-file=/etc/kubernetes/audit-policy.yaml --proxy-client-key-file=/etc/kubernetes/ssl/kube-apiserver-proxy-client-key.pem --service-account-lookup=true --bind-address=0.0.0.0 --audit-log-path=/var/log/kube-audit/audit-log.json --audit-log-format=json --tls-cert-file=/etc/kubernetes/ssl/kube-apiserver.pem --tls-private-key-file=/etc/kubernetes/ssl/kube-apiserver-key.pem --kubelet-client-certificate=/etc/kubernetes/ssl/kube-apiserver.pem --service-account-key-file=/etc/kubernetes/ssl/kube-service-account-token-key.pem --service-account-issuer=rke --kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname --profiling=false --audit-log-maxbackup=10 --requestheader-allowed-names=kube-apiserver-proxy-client --etcd-servers=https://172.31.6.132:2379 --kubelet-client-key=/etc/kubernetes/ssl/kube-apiserver-key.pem --proxy-client-cert-file=/etc/kubernetes/ssl/kube-apiserver-proxy-client.pem --service-node-port-range=30000-32767 --authentication-token-webhook-cache-ttl=5s --service-account-signing-key-file=/etc/kubernetes/ssl/kube-service-account-token-key.pem --requestheader-extra-headers-prefix=X-Remote-Extra- --etcd-cafile=/etc/kubernetes/ssl/kube-ca.pem --etcd-certfile=/etc/kubernetes/ssl/kube-node.pem --anonymous-auth=false --advertise-address=172.31.13.71 --tls-cipher-suites=TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305 --enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,NodeRestriction,Priority,TaintNodesByCondition,PersistentVolumeClaimResize --etcd-keyfile=/etc/kubernetes/ssl/kube-node-key.pem --secure-port=6443 --authentication-token-webhook-config-file=/etc/kubernetes/kube-api-authn-webhook.yaml --allow-privileged=true --api-audiences=unknown --client-ca-file=/etc/kubernetes/ssl/kube-ca.pem --etcd-prefix=/registry --audit-log-maxsize=100 --requestheader-client-ca-file=/etc/kubernetes/ssl/kube-apiserver-requestheader-ca.pem --authorization-mode=Node,RBAC
+```
+
+### 1.2.25 Ensure that the --service-account-key-file argument is set as appropriate (Automated)
+
+
+**Result:** pass
+
+**Remediation:**
+Edit the API server pod specification file /etc/kubernetes/manifests/kube-apiserver.yaml
+on the control plane node and set the --service-account-key-file parameter
+to the public key file for service accounts. For example,
+--service-account-key-file=<filename>

```

Comment:
```suggestion
`--service-account-key-file=<filename>`

```

Converting to inline code since Docusaurus will throw an error because it treats `<filename>` as an HTML tag.